tests/Resources/ODS.robot

```diff
@@ -202,13 +202,23 @@ Verify Default Access Groups Settings
         Verify RHODS Groups Config Map Contains Expected Values   &{exp_values}
     END
 
-Disable Access To Grafana Using OpenShift Port Forwarding
+Kill Process
     [Documentation]   Kill process running in background based on Id
-    [Arguments]  ${PROC}
-    Terminate Process   ${PROC}
+    [Arguments]  ${proc}
+    Terminate Process   ${proc}
```

Comment:
Actually, I'm not sure it makes sense to have this as a keyword at all, you're simply wrapping the call to `Terminate Process` in the `Kill Process` keyword.

Reply:
Updated